databroker/_drivers/mongo_embedded.py

```diff
@@ -225,12 +226,23 @@ def __init__(self, datastore_db, *, handler_registry=None, root_map=None,
         else:
             self._db = datastore_db
         self._query = query or {}
+        self._root_map = root_map
+        self._filler_class = filler_class
+        self._delayed_filler_class = delayed_filler_class
 
         super().__init__(handler_registry=handler_registry,
                          root_map=root_map,
                          filler_class=filler_class,
                          **kwargs)
 
+    def _get_filler(self):
+        return self._filler_class(
+                self._handler_registry, root_map=self._root_map, inplace=False)
+
+    def _get_delayed_filler(self):
```

Comment:
This method and at least one other like it in other modules survived our purge during the rebase.

Reply:
Re-upping this

Reply:
updated